app/jobs/invoices/provider_taxes/pull_taxes_and_apply_job.rb

```diff
@@ -8,7 +8,7 @@ class PullTaxesAndApplyJob < ApplicationJob
       retry_on BaseService::ThrottlingError, wait: :polynomially_longer, attempts: 25
 
       def perform(invoice:)
-        Invoices::ProviderTaxes::PullTaxesAndApplyService.call!(invoice:)
+        Invoices::ProviderTaxes::PullTaxesAndApplyService.call(invoice:)
```

Comment:
This is actually swallowing the error. It feels a bit risky.

Shouldn't we handle the error cases in the service to avoid loosing important errors?

Reply:
Maybe I am missing something, but with new logic, all errors related to taxes will be stored in the DB and user can handle them. Everything else will be raised since we don't rescue anymore 🤔

Reply:
One more update:)